src/ostorlab/runtimes/local/runtime.py

```diff
@@ -550,3 +553,29 @@ def describe_vuln(self, scan_id: int, vuln_id: int):
             console.success('Vulnerabilities listed successfully.')
         except sqlalchemy.exc.OperationalError:
             console.error('Vulnerability / scan not Found.')
+
+    def dump_vulnz(self, scan_id: int, output: str, output_format: str):
+        """Dump found vulnerabilities of a scan in a specific format."""
+        database = models.Database()
+        session = database.session
+        severity_sort_logic = case(value=models.Vulnerability.risk_rating,
+                                   whens=risk_rating.RATINGS_ORDER).label('severity')
+        vulnerabilities = session.query(models.Vulnerability).filter_by(scan_id=scan_id). \
+            order_by(severity_sort_logic).all()
+
+        vulnz_list = []
+        for vulnerability in vulnerabilities:
+            vuln = {
+                'id':vulnerability.id,
+                'risk_rating': vulnerability.risk_rating.value,
+                'cvss_v3_vector': vulnerability.cvss_v3_vector,
+                'title': vulnerability.title,
+                'short_description': vulnerability.short_description
+            }
+            vulnz_list.append(vuln)
+        if output_format == 'csv':
```

Comment:
This is not a great api design since every time we add format, we would need to change the API to add it.

A better impelement is that all API return a list in a specific format and that list get passed to the selected dumper class, this way it is agnostic to the format.